includes/reader-revenue/woocommerce/class-woocommerce-connection.php

```diff
@@ -266,6 +266,19 @@ public static function force_disable_order_attribution( $should_allow ) {
 	 * @return bool
 	 */
 	public static function send_customizable_receipt_email( $enable, $order, $class ) {
+		// If there are no donation products in the order, do not override the default WC receipt email.
+		$has_donation_product = false;
```

Comment:
In other places we check this using 

```
$product_id = \Newspack\Donations::get_order_donation_product_id( $order_id );
```

which is basically the same thing you do here. Isn't it simples if we just use that?

Reply:
Huh, thanks! Changed in b3e62f020de78e416dd5efaa905a7c2ee6ee09bb